chambersc/encounters/mpg_subversion.lua

```diff
@@ -42,6 +39,11 @@ function Subversion_Say(e)
     e.self:Say("This is the Mastery of Subversion trial. You must demonstrate your ability to think on your feet, to defeat an unbeatable opponent. Are you ready to [" .. eq.say_link("begin", false, "begin") .. "]?");
 
   elseif (e.message:findi("begin")) then
+    local dz = eq.get_expedition()
+    if dz.valid then
+      dz:SetLocked(true, ExpeditionLockMessage.Begin, 14) -- live uses "Event Messages" type 365 (not in emu clients)
```

Comment:
Love the constant usage

Reply:
I have it on my list to do a PR for the missing color constants as well (e.g. Color.Lime) unless someone else gets to it